components/style/properties/shorthand/box.mako.rs

```diff
@@ -6,19 +6,40 @@
 
 <%helpers:shorthand name="overflow" sub_properties="overflow-x overflow-y"
                     spec="https://drafts.csswg.org/css-overflow/#propdef-overflow">
-    use properties::longhands::{overflow_x, overflow_y};
+    use properties::longhands::overflow_x;
```

Comment:
Perhaps, given we use it for both, `use properties::longhands::overflow_x::SpecifiedValue as overflow`?